ERCS/erc-7715.md

```diff
@@ -96,20 +90,14 @@ An array of `PermissionRequest` objects is the final `params` field expected by
                 address:'0x016562aA41A8697720ce0943F003141f5dEAe006',
             }
         },
-        permission: {
+        permissions: [
+          {
             type: 'native-token-transfer',
             data: {
                 allowance: '0x1DCD6500'
             }
-        },
-        policies: [
-            {
-                type: 'gas-limit',
-                data: {
```

Comment:
Can you bring back the gas-limit in this example? So we could see it used in parallel with the transfer permission?

Reply:
done!